websitetoolbox.com.forum.json

```diff
@@ -0,0 +1,30 @@
+{
+  "providerId":"websitetoolbox.com",
```

Comment:
I would consider if removing syncPubKeyDomain from this template is wise.  Typically a static template (like the other one) is somewhat safe to apply without a signature.

But you have a variable in a host name here.  If I want to mess with you, I generate a link that applies this template with a random v1 variable;  and this would potentially overwrite and hijack a random sub-domain in the user's domain. Of course, it would point to your forum application.  So it is probably OK to leave out.

Just thinking out loud.

Depending on what @pawel-kow says about the variable in the host, we could publish this template.  I supsect the DNS Providers would onboard despite and allow the variable in the host....

Reply:
Thanks, that's a good point. We omitted it initially just to keep things as simple as possible while setting it up with one DNS provider. Once things are working properly, perhaps we could make a modification and add in syncPubKeyDomain before rolling out to other DNS providers or going live.